kf4cic.c

```diff
@@ -227,6 +227,31 @@ EFI_STATUS load_and_start_tos(EFI_HANDLE image)
 
 	ret = start_trusty(tosimage);
 	debug(L"start_trusty return: %r(%x)\n", ret, ret);
+
+	return ret;
+}
+
+static VOID EFI_API activate_vtd_vmcall(EFI_EVENT event, VOID *ctx)
+{
+	(void)event;
```

Comment:
is it a common way? or, use UNUSED flag?

Reply:
a common practice when you do not want to add UNUSED flag.